packages/web/hooks/use-swap.ts

```diff
@@ -531,3 +505,95 @@ function useSwapAsset(
     isLoading: isLoading && !existingAsset,
   };
 }
+
+/** Iterates over available and identical routers and sends input to each one individually.
+ *  Results are reduced to best result by out amount.
+ *  Also returns the number of routers that have fetched and errored. */
+function useQueryBestQuote(
```

Comment:
What are your thoughts on renaming this function to `useRouterBestQuote`?

Reply:
I like this, but I'll include "query" since it's a wrapper around `useQueries`